addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15419,13 +15419,13 @@ msgstr ""
 #. Description of settings section "System" with label #13000
 #: system/settings/settings.xml
 msgctxt "#36349"
-msgid "Section that contains the System related settings for the device this application is installed on."
+msgid "Section that contains the system related settings for the device this application is installed on."
 msgstr ""
 
 #. Description of setting "System -> Power saving -> Try to wake remote servers on access" with label #13026
 #: system/settings/settings.xml
 msgctxt "#36350"
-msgid "Automatically send 'Wake-On-Lan' to server(s) right before trying to access shared files or services."
+msgid "Automatically send 'Wake-on-Lan' to server(s) right before trying to access shared files or services."
```

Comment:
Actually no as you pointed out yourself earlier
http://en.wikipedia.org/wiki/Wake-on-LAN